doc/external_resources.md

```diff
@@ -0,0 +1,21 @@
+# External Resources
+
+Below is a curated collection of external PyGMT resources.
+
+*To add your contribution to this collection, follow [these instructions](https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#editing-the-documentation)
+to submit a pull request with your recommended addition to the
+[External Resources file](https://github.com/GenericMappingTools/pygmt/blob/master/doc/external_resources.md).*
+
+---
+
+## Tutorials
+
+- [2021 PyGMT course at the UAF Geophysical Institute](https://github.com/liamtoney/gi-pygmt-2021) by Liam Toney
+- [Remote Online Sessions for Emerging Seismologists (ROSES): Unit 8 - PyGMT](https://www.iris.edu/hq/inclass/lesson/728) by Liam Toney
+- [PyGMT Tutorial in 2021](https://github.com/MIGG-NTU/PyGMT2021) by MIGG-NTU
+- [PyGMT Workshop at FOSS4G Oceania 2019](https://github.com/GenericMappingTools/foss4g2019oceania) by Wei Ji
+
+## Examples
```

Comment:
Could we change this to be a bit more descriptive? Something like "Examples from Publications and Posters", or something along those lines?

Reply:
Done in 1a1670759b4a48b49558d93c5052cdcf7e326b80.